lib/resources/r4/immunization.yaml

```diff
@@ -150,6 +150,7 @@ fields:
         "reference": "Encounter/36209678"
       }
     }
+  note: encounter is required field to create charted and staged immunizations
```

Comment:
you are not creating immunizations here. Updating. 
FHIR does not support updateing staged immunization.
This note is not necessary. can be removed. Rest looks good